charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -23,7 +23,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 67.0.0
+version: 68.0.0
```

Comment:
```suggestion
version: 67.1.0
```

I think a minor bump should be enough

Reply:
Thanks for your adjustment! I wasn't sure so I wanted to play it safe. :)